### ISSUE TITLE
Make page title and logo customizable

### ISSUE THREAD

**Issue by TwinProduction** [TwinProduction/gatus#77]
As nice as "Health Dashboard" is, I believe people may be interested in having a custom title.

The configuration should look like this:
```yaml
web:
  title: Example's Health Dashboard
```

If `title` is not set, the default title should be `Health Dashboard | Gatus`.

**Comment by mharjac** [comment]
It would be also nice if you add option for using custom logo next to the title, and to keep project visibility, you could put something like "Powered by Gatus" in the footer.

**Comment by AndreiPaulau** [comment]
Definitely,
Custom title, favicon, logo to the right of page are cool features! Woted!

**Comment by TwinProduction** [comment]
Note to self: could implement this by using a template for `index.html` and parsing the values like this:
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>{{ .Title }}</title>
    ...
  </head>
  <body>
    ...
    <script>localStorage.setItem('logo', "{{ .Logo }}');</script>
    ...
  </body>
</html>
```

**Comment by TwinProduction** [comment]
This was long overdue, but at last.. 
![image](https://user-images.githubusercontent.com/15699766/132940412-3ecc144a-db6e-443f-b0b6-d7e6f393559e.png)

I'll be merging it in master soon.

**Comment by TwinProduction** [comment]
Released in [v3.1.0](https://github.com/TwinProduction/gatus/releases/tag/v3.1.0)
